systori/apps/task/serializers.py

```diff
@@ -224,13 +236,13 @@ def perform(self):
 
 
 class JobSerializer(serializers.ModelSerializer):
-    groups = GroupSerializer(many=True, required=False)
-    tasks = TaskSerializer(many=True, required=False)
+    groups = GroupSerializer(required=False, many=True)
+    tasks = TaskSerializer(required=False, many=True)
```

Comment:
These are unnecessary when looked at the git diff, but they happened over the course of multiple commits and experimentation, so no easy way of reversing those commits.

Reply:
lol, ok, np